code/game/gamemodes/cult/runes.dm

```diff
@@ -313,8 +314,9 @@ structure_check() searches for nearby cultist structures required for the invoca
 
 	rune_in_use = TRUE
 	var/mob/living/L = pick(offer_targets)
-	if(L.mind in GLOB.sacrificed)
-		fail_invoke()
+	if(HAS_TRAIT(L, TRAIT_CULT_IMMUNITY))
+		fail_invoke(L)
+		to_chat("<spacn class='warning'>This sacrifice was already converted recently. Wait a moment before trying again!</span>")
```

Comment:
you misspelt span
also, to_chat needs a target arg. 
also `fail_invoke` says this exact message, that you copy pasted, because you misspelt span in that one aswell.